modules/KalturaSupport/resources/uiConfComponents/titleLabel.js

```diff
@@ -23,6 +23,25 @@
 						.addClass('truncateText');
 				}
 			});
+
+			this.bind('updateLayout', function(){
+					var availableWidth = _this.getAvailableWidth(); // available width for title including buttons space and extra space for clarity
+					if (_this.getConfig('truncateLongTitles') && _this.getComponent().width() >= availableWidth) {
+						_this.getComponent()
+							.attr('title', _this.getConfig('text'))
+							.width(availableWidth)
+							.addClass('truncateText');
+					} else {
+						_this.getComponent()
+							.attr('title', _this.getConfig('text'))
+							.width(availableWidth)
+							.removeClass('truncateText');
+					}
+				}
+			);
+		},
+		getAvailableWidth:function(){
+			return this.embedPlayer.getWidth() - ($('.topBarContainer .btn').length + 1) * 30;
```

Comment:
@itaykinnrot - maybe instead of hardcoded 30 use jQuery width() on a .btn selector? The "theme" plugin allows resizing buttons...